src/test/java/seedu/address/model/person/PredicateContainerTest.java

```diff
@@ -147,5 +82,51 @@ public void test_toString() {
         assertEquals(expected, actual);
     }
 
+    public PredicateContainer getPredicateContainer1() {
```

Comment:
Could add comments explaining what getPredicateContainter 1 and 2 do

Reply:
Added!